### ISSUE TITLE
Blazor Mobile Bindings

### ISSUE THREAD

**Issue by Michael-Burschel** [thudugala/Plugin.LocalNotification#193]
Hi,

I tried the notifications in my Xamarin App and it works fine.
Recently I started to work on some Blazor Projects, in specific I tried the notifications for a Blazor Mobile Bindings App. I can configure it but I cannot get it working somehow. No notification is showing up at all.
Is it supposed to be working or is it not working at all for Blazor?

**Comment by thudugala** [comment]
@Michael-Burschel can attach a sample project?

**Comment by Michael-Burschel** [comment]
Thanks for your reply.
Surprisingly it is working now, but for ne specific reason.
I tried it without any Plugin --> https://www.youtube.com/watch?v=Lqhg9Lf8JP4
It worked perfectly and I tried your plugin again. Suddenly everything worked as expected without any changes. I tried it on multiple devices as well. There was some kind of magic involved to nag me! ;-) 

I will close this because everything is working. Thanks and sorry!
